typescript/cli/src/commands/relayer.ts

```diff
@@ -1,35 +1,82 @@
-import { HyperlaneCore, HyperlaneRelayer } from '@hyperlane-xyz/sdk';
+import {
+  ChainMap,
+  HyperlaneCore,
+  HyperlaneRelayer,
+  RelayerCacheSchema,
+} from '@hyperlane-xyz/sdk';
 
 import { CommandModuleWithContext } from '../context/types.js';
 import { log } from '../logger.js';
+import { tryReadJson, writeJson } from '../utils/files.js';
+import { selectRegistryWarpRoute } from '../utils/tokens.js';
 
-import { agentTargetsCommandOption } from './options.js';
+import {
+  agentTargetsCommandOption,
+  overrideRegistryUriCommandOption,
+  symbolCommandOption,
+} from './options.js';
 import { MessageOptionsArgTypes } from './send.js';
 
+const DEFAULT_RELAYER_CACHE = `${overrideRegistryUriCommandOption.default}/relayer-cache.json`;
```

Comment:
Could this maybe lead to someone accidentally committing this cache file (in certain cases)?

Reply:
definitely